src/pages/students/Students.tsx

```diff
@@ -85,7 +85,15 @@ function Students() {
       <div className="form-group">
         <div className="column1">
           <label>DNI:</label>
-          <input {...register("dni")} type="number" />
+          <input
+            {...register("dni")}
+            type="text"
```

Comment:
Acá te quedó como text en vez de number


```suggestion
            type="number"
```